HW2_Bredov/calculator.py

```diff
@@ -0,0 +1,25 @@
+def divide(a, b):
+  return a/b
+
+def add(a,b):
+    return a+b
```

Comment:
Здесь и в других местах не хватает оформления пробелами. Всё правильно, но на будущее вот так будет выглядеть чуть лучше:
```suggestion
def add(a, b):
    return a + b
```